spectrocloud/resource_cluster_libvirt.go

```diff
@@ -487,6 +514,23 @@ func flattenMachinePoolConfigsLibvirt(machinePools []*models.V1LibvirtMachinePoo
 	return ois
 }
 
+func flattenGpuDevice(gpus []*models.V1GPUDeviceSpec) []interface{} {
```

Comment:
Let's write a unit test for this functionality.

Reply:
added unit test